rb/lib/selenium/webdriver/chrome/options.rb

```diff
@@ -74,12 +64,21 @@ class Options < WebDriver::Options
         # @option opts [Array<String>] :window_types A list of window types to appear in the list of window handles
         #
 
-        def initialize(profile: nil, encoded_extensions: nil, **opts)
+        def initialize(profile: nil, encoded_extensions: [], **opts)
           super(**opts)
 
           @profile = profile
-          @options[:encoded_extensions] = encoded_extensions if encoded_extensions
-          @options[:extensions]&.each(&method(:validate_extension))
+          @options[:encoded_extensions] = encoded_extensions
+
+          @options[:args] ||= []
```

Comment:
oh, sure, it just always looks weird to me to add defaults that way, but it's probably a good change since it will also decrease cyclomatic complexity.